app/services/event.service.js

```diff
@@ -284,48 +295,48 @@
         if(filteredList.data[x].date.length > 1){
           // loop through the dates of the repeating events and pull out the object for it to loop of it with the index
           filteredList.data[x].date.forEach(function(n){
-            // Check Start Date and End Date. Only needed for All filter
-            var start = new Date(filterString.split('filter[date][value][0]=')[1].split('&')[0].split(' ')[0]).getTime() / 1000;
-            if( filterString.includes('filter[date][value][1]=') ){          
-              var end = new Date(filterString.split('filter[date][value][1]=')[1].split('&')[0].split(' ')[0]).getTime() / 1000;
-              if(n.start_unix > start && n.end_unix < end){
+            if(!data[n.start_unix]){
+              data[n.start_unix] = [];
+            }
+            // if there is an end date and started and hasn't ended
+            if( filterString.includes('filter[date][value][1]=') && n.start_unix > start && n.end_unix < end){
                 var copy = Object.assign({}, filteredList.data[x]);  // make hard copy of this object
                 copy.item = z;     //give the index for the calendar
-                data[n.start_unix] = copy;     
-              } 
-            }else{
-              if(n.start_unix > start){    
-                var copy = Object.assign({}, filteredList.data[x]);
-                copy.item = z;
-                data[n.start_unix] = copy;  
-              }  
+                data[n.start_unix].push(copy);     
+            // if has started
+            }else if(n.start_unix > start){    
+              var copy = Object.assign({}, filteredList.data[x]);
```

Comment:
`Object.assign` is not available in IE. I believe we have jQuery available so you could use `var copy = jQuery.extend(true, {}, filteredList.data[x])` instead.